src/data/ACTIONS/SAM.js

```diff
@@ -0,0 +1,240 @@
+// Samurai Actions
+export default {
+	//-----
+	//Player GCDs
+	//-----
+	HAKAZE: 
```

Comment:
Not sure if `eslint` picks this up, but please follow the convention in the rest of the codebase for this, e.g.:

```js
export default {
	HAKAZE: { // on same line
		key: value, // space between key and val
	},
}
```